WordPress/src/main/java/org/wordpress/android/ui/people/utils/PeopleUtils.java

```diff
@@ -46,8 +49,13 @@ public void onErrorResponse(VolleyError volleyError) {
             }
         };
 
+        Map<String, String> params = new HashMap<>();
+        params.put("number", Integer.toString(PeopleUtils.FETCH_USERS_LIMIT));
```

Comment:
Guys, heads up. Not sure if we're hitting the exact same endpoint, or there's a difference. But on my end, the `number` parameter appears to be disregarded.

Instead, i'm using `page`. (Checked with dotcom source!).